<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>emp</class>
 <widget class="QDialog" name="emp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>890</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit{
border-radius:10px;
}

QPushButton{
    
	background-color: rgb(188, 188, 188);
	border-radius:20px;
	
	font: 9pt &quot;MS Reference Sans Serif&quot;;

}

#widget{
background-color: QLinearGradient(x1: 0,y1: 0,x2: 0,y2: 1,stop: 0 #e66465, stop:1 #9198e5);
	border-radius:20px;
}
#widget_2{
	
	border-radius:20px;
	background-color: rgb(255, 255, 255);
}

QLineEdit{
	
	 
	background-color: rgb(204, 204, 204);
   
}

QLabel{
	
	font: 11pt &quot;MV Boli&quot;;
}
#widget_3
{
border-radius:20px;
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>1461</width>
     <height>801</height>
    </rect>
   </property>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>140</y>
      <width>741</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>501</width>
      <height>731</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="le_nom">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_prenom">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_id">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>250</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_cin">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>340</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="la_fonctionnalites">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>430</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_grade">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>510</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="modifier">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>590</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>modifier</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>nom:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>140</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>prénom:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>230</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>identifiant:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>320</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>cin:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>410</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>fonctionalités:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>480</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>grade:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>660</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>590</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>660</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>afficher</string>
     </property>
    </widget>
    <widget class="QPushButton" name="valider">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>227</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
background-color: rgb(153, 153, 229);</string>
     </property>
     <property name="text">
      <string>valider</string>
     </property>
    </widget>
    <widget class="QPushButton" name="annuler">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>380</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
background-color: rgb(227, 0, 0);</string>
     </property>
     <property name="text">
      <string>annuler</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="recherche">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>60</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="trier_par">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>100</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>trier par:</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>nom</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>prenom</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="chercher_par">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>60</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>chercher par :</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>nom</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>prénom</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="ordre_tri">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>100</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>ordre de tri:</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>croissant</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>décroissant</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>550</y>
      <width>331</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="stat">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>17</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;</string>
    </property>
    <property name="text">
     <string>stat</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>1121</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 0, 0)
</string>
   </property>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_10">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);

color: rgb(0, 0, 0);

font: 14pt &quot;MV Boli&quot;;</string>
    </property>
    <property name="text">
     <string>Gestion des employés</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>chat</string>
    </property>
    <property name="icon">
     <iconset resource="style.qrc">
      <normaloff>:/new/prefix1/chat1.png</normaloff>:/new/prefix1/chat1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>pdf</string>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>40</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="style.qrc"/>
 </resources>
 <connections/>
</ui>
